edgePy/util.py

```diff
@@ -0,0 +1,23 @@
+""" Utilities to support functions and classes """
+import logzero  # type: ignore
+
+
+def _setup_log():
```

Comment:
Could we parametrize this with `name` so that we can have different loggers for the main interactive library and for scripts?

https://logzero.readthedocs.io/en/latest/#custom-logger-instances